Platform/CoffeelakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1041,6 +1042,14 @@ BoardInit (
   case PrePciEnumeration:
     break;
   case PostPciEnumeration:
+    if (GetPayloadId () == UEFI_PAYLOAD_ID_SIGNATURE && GetBootMode() == BOOT_ON_S3_RESUME) {
+      //
+      // Clear Smi and restore S3 regs on S3 resume
+      //
+      ClearSmi ();
+      RestoreS3RegInfo (FindS3Info (S3_SAVE_REG_COMM_ID));
+      TriggerBlSwSmi (); 
+    }
     break;

```

Comment:
Why have to move to PostPciEnumeration?

Reply:
because we cannot trigger smi after mp init done

Reply:
do you know the reason why we can't trigger smi after MP init done?